mergestat/mergestat
# feat(import): Switch From GitHub GraphGQL API to REST API for auto imports

This pr changes the Graphql API approach  for  the REST API  on the auto imports feature . This way  we can import repos without  a GitHub token .There are some sync that won't work without proper validation, those are :

1. PR Commits
2. Pr Reviews
3. Repo Issues
4. Repo metadata
5. Pull Requests
6. Repo Stars

In this content we also reintroduce stdout logging to the importer

To test this  you need to 

1. Log into mergestat
8. go to /repos page and click on  add repositories
9. Choose an user/org with public repos 
10. Those repos should be appearing  after a while

Closes #540
